packages/tsconfig/nextjs.json

```diff
@@ -4,9 +4,11 @@
   "extends": "./base.json",
   "compilerOptions": {
     "plugins": [{ "name": "next" }],
+		"moduleResolution": "node",
```

Comment:
nit: indentation.
Wouldn't happen with VS Code 😆

Reply:
really need to update my vim setup istg